mutiny-core/src/nodemanager.rs

```diff
@@ -812,7 +821,7 @@ impl NodeManager {
             .output
             .iter()
             .find_map(|o| {
-                if let Ok(addr) = Address::from_script(&o.script_pubkey, self.network) {
+                if let Ok(addr) = Address::from_script(&o.script_pubkey, used_network) {
```

Comment:
This happens in two more places in `onchain.rs`, is that needed there?

Reply:
No, those are correct. This is only a problem here because they are coming from parsing json. In `onchain.rs` we want to use the proper network so we can compare.

Reply:
wait actually we might want to do it

Reply:
I am just gonna make address labels use a string